db/keyval/bolt/plugin_impl_bolt.go

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bolt
+
+import (
+	"github.com/boltdb/bolt"
+	"github.com/ligato/cn-infra/datasync/resync"
+	"github.com/ligato/cn-infra/db/keyval"
+	"github.com/ligato/cn-infra/db/keyval/kvproto"
+	"github.com/ligato/cn-infra/flavors/local"
+	"log"
+)
+
+// Config represents configuration for Bolt plugin.
+type Config struct {
+	DbPath          string `json:"db-path"`
+	BucketSeparator string `json:"bucket-separator"`
+}
```

Comment:
add config items to define 
- `FilePermissions` of created db
- `timeout` for opening db